include/clang/CrossTU/CrossTranslationUnit.h

```diff
@@ -168,6 +178,7 @@ class CrossTranslationUnitContext {
 
   llvm::StringMap<std::unique_ptr<clang::ASTUnit>> FileASTUnitMap;
   llvm::StringMap<clang::ASTUnit *> FunctionASTUnitMap;
+  llvm::StringMap<clang::ASTUnit *> SrcFileASTUnitMap;
```

Comment:
Why can't we use `FileASTUnitMap`?
Please include the answer as a comment above both member variables: `FileASTUnitMap` and `SrcFileASTUnitMap`.

Reply:
It can be possible to get the source filename from AST filename but it is more complicated and may not work in all cases or if the AST file naming is changed later.

Reply:
OK